src/components/Footer/index.tsx

```diff
@@ -136,6 +136,7 @@ const StyledAddress = styled.address`
 
 type FooterPropsType = {
   theme?: ThemeType;
+  copyText?: string;
```

Comment:
we could make it a bit more flex able if this was 

copyText?: ReactNode;

That way the whole middle section is customisable some could chucking their own component with a button or whatever in the future

 ```typeof props.copyText === 'string' ? <Text textSize="s">{props.copyText}</Text> : <>{props.copyText}</>```